README.md

```diff
@@ -130,6 +130,28 @@ Output:
 1
 ```
 
+Full Damerau-Levenshtein matching can be used if you want to consider character transpositions as a valid edit operation.
+
+```javascript
+console.log(natural.LevenshteinDistance("az", "za", { damerau: true }));
```

Comment:
would not be better to use ```natural.damarauDistance(...)``` ?

Reply:
yea i agree with @syzer seems like it would be easier to use that way. @reblws would you be able to quickly update that?

Reply:
Yeah, no problem. I'll put this in a separate file, along with an extra option for simpler Optimal String Alignment.

I feel a bit uneasy about pasting functions, namely `getMinCostSubstring` and `_getMatchStart` from `levenshtein_distance.js`. I'll just copy & paste them for now, but let me know if you want them to be in a common module.

Also, last thing: should the function be called `DamerauLevenshteinDistance` or `DamerauDistance`?

Reply:
hmm yea sorry i forgot how this was all structured, we could change the way it exports snd have the one file export both, then update [this](https://github.com/NaturalNode/natural/blob/master/lib/natural/index.js#L78) to expose both. does that make sense?

Reply:
Ok, so we'll export both `DamerauLevenshteinDistance` and `LevenshteinDistance` from `levenshtein_distance.js` and then update `natural/index.js` to import from `levenshtein_distance` explicitly?

Reply:
Ok, all done